### ISSUE TITLE
Improve lore2sphinx buildbot results for `projects/conch`

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4566]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)| @khorn reported|
|-|-|
|Trac ID|trac#4566|
|Type|enhancement|
|Created|2010-07-11 06:34:35Z|
|Branch|https://github.com/twisted/twisted/tree/improve-lore2sphinx-output-for-projects-conch-4566|

Make the lore2sphinx buildbot results for `projects/conch` look as good as possible by modifying the lore sources.

Things to look for:
 1. nested inline markup should be removed
 1. whitespace should be added or removed as needed to make ReST markup render properly
 1. anything else that you can improve by modifying the Lore sources

Note that you probably want to run lore2sphinx on a local copy for development purposes.

You can get lore2sphinx from:
http://bitbucket.org/khorn/lore2sphinx

Attachments:

* [lore-conch-cleanups.diff](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/31f/31f18ad59e1fa08510b5cfba409255a9ecfc85e7/7ba94e0300963dbf5ce74f9add2d66716348eca9.diff) (5542 bytes) - added by Screwtape on 2010-07-11 06:40:06Z - A patch that cleans up the Sphinx rendering of the Conch docs.

<details><summary>Searchable metadata</summary>

```
trac-id__4566 4566
type__enhancement enhancement
reporter__khorn khorn
priority__normal normal
milestone__Lore_to_Sphinx Lore to Sphinx
branch__branches_improve_lore2sphinx_output_for_projects_conch_4566 branches/improve-lore2sphinx-output-for-projects-conch-4566
branch_author__screwtape screwtape
status__closed closed
resolution__fixed fixed
component__core core
keywords__documentation documentation
time__1278830075000000 1278830075000000
changetime__1279684286000000 1279684286000000
version__None None
owner__Screwtape Screwtape

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Screwtape set owner to Screwtape<br>Screwtape set status to `assigned`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="khorn's avatar" src="https://avatars.githubusercontent.com/u/815352?s=50" width="50" height="50">](https://github.com/khorn)<a name="note_2"></a>|@khorn commented|
|-|-|

The lore2sphinx buildbot results can be found at:

http://buildbot.twistedmatrix.com/builds/sphinx-html-15615/

(or at different directories for different revisions...the containing directory is browseable)

**Comment by twisted-trac** [comment]
|<img alt="Screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|Screwtape removed owner<br>Screwtape set status to `new`|
|-|-|

Changes in this patch:

 * Added trailing spaces to a few lines so that the element on the next line would be correctly rendered, not rendered as raw ReST markup.
 * Fixed some links to point to the correct places.
 * The link to `telnet_echo.tac` now shows its description.
 * Added descriptions for `sshsimpleclient.py` and `sshsimpleserver.py`.
 * Filenames and descriptions on the examples page are separated by hyphens, as they are in the Twisted Core examples page

**Comment by twisted-trac** [comment]
|<img alt="Screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_4"></a>|Screwtape commented|
|-|-|

Oh, and things that aren't fixed: spaces between inline markup and end-of-sentence punctuation.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_5"></a>|@exarkun set owner to Screwtape|
|-|-|

Looks fine, although it's missing a news fragment.  One thing I noticed while reviewing this, though, is that code listings no longer have line numbers.  Could you file a ticket for that?

**Comment by twisted-trac** [comment]
|<img alt="Screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_6"></a>|Screwtape commented|
|-|-|

> One thing I noticed while reviewing this, though, is that code listings no longer have line numbers.
Filed as [#4586](https://github.com/twisted/twisted/issues/4586).

**Comment by twisted-trac** [comment]
|<img alt="screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_7"></a>|screwtape commented|
|-|-|

(In [29822]) Create branch improve-lore2sphinx-output-for-projects-conch-4566

**Comment by twisted-trac** [comment]
|<img alt="screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_8"></a>|screwtape commented|
|-|-|

(In [29823]) Apply lore-conch-cleanups.diff

refs [#4566](https://github.com/twisted/twisted/issues/4566)

**Comment by twisted-trac** [comment]
|<img alt="screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_9"></a>|screwtape commented|
|-|-|

(In [29824]) Address exarkun's code-review comments.

refs [#4566](https://github.com/twisted/twisted/issues/4566)

**Comment by twisted-trac** [comment]
|<img alt="Screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_10"></a>|Screwtape removed owner|
|-|-|

Sphinx output [here](http://buildbot.twistedmatrix.com/builds/sphinx-html-15632/).

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_11"></a>|@exarkun set owner to Screwtape|
|-|-|

Looks good.  Please merge.

**Comment by twisted-trac** [comment]
|<img alt="screwtape's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_12"></a>|screwtape set status to `closed`|
|-|-|

(In [29850]) Merge improve-lore2sphinx-output-for-projects-conch-4566: Tidy conch docs for lore2sphinx conversion.

Author: screwtape
Reviewers: exarkun
Fixes: [#4566](https://github.com/twisted/twisted/issues/4566)

 - Added trailing spaces to a few lines so that the element on the next
   line would be correctly rendered, not rendered as raw ReST markup.
 - Fixed some links to point to the correct places.
 - The link to telnet_echo.tac now shows its description.
 - Added descriptions for sshsimpleclient.py and sshsimpleserver.py.
 - Filenames and descriptions on the examples page are separated by
   hyphens, as they are in the Twisted Core examples page
